src/main/java/com/ericsson/ei/subscription/SubscriptionValidator.java

```diff
@@ -59,127 +60,143 @@ private SubscriptionValidator() {
      * @param subscription
      * @throws SubscriptionValidationException
      */
-    public static void validateSubscription(Subscription subscription) throws SubscriptionValidationException {
-        LOGGER.debug("Validation of subscription " + subscription.getSubscriptionName() + " Started.");
-        validateSubscriptionName(subscription.getSubscriptionName());
-        validateNotificationMessageKeyValues(subscription.getNotificationMessageKeyValues(),
-                subscription.getRestPostBodyMediaType());
-        validateNotificationType(subscription.getNotificationType());
-        validateNotificationMeta(subscription.getNotificationMeta(), subscription.getNotificationType());
-        if (subscription.getNotificationType().equals("REST_POST")) {
-            validateRestPostMediaType(subscription.getRestPostBodyMediaType());
-        }
-        validateWithSchema(subscription);
-        LOGGER.debug("Validating of subscription " + subscription.getSubscriptionName() + " finished successfully.");
+    public static void validateSubscription(Subscription subscription)
+	    throws SubscriptionValidationException {
+	LOGGER.debug("Validation of subscription {} Started.",
+		subscription.getSubscriptionName());
+	validateSubscriptionName(subscription.getSubscriptionName());
```

Comment:
Maybe switch order of this method call to validate the name of the subscription before the log message is printed. This way we won't get null in the console.

Reply:
Done.